Simplified/NYPLBookDetailDownloadingView.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)init
   self.percentageLabel.font = [UIFont systemFontOfSize:14];
   self.percentageLabel.textColor = [NYPLConfiguration backgroundColor];
   self.percentageLabel.textAlignment = NSTextAlignmentRight;
-  self.percentageLabel.text = @"0%";
+  self.percentageLabel.text = NSLocalizedString(@"PercentageLabelZero", nil);
```

Comment:
It probably doesn't make sense to localize numbers like this, especially since this percentage is going to be dynamically updated. Instead, I would suggest using a wrapper function that explicitly disables the localization warning via `__attribute__((annotate("returns_localized_nsstring")))`. We can write that function once—maybe call it `NYPLDoNotLocalize`—and reuse it. See [Clang's documentation](https://clang-analyzer.llvm.org/faq.html#unlocalized_string) for more information.

Reply:
@winniequinn what file would be a good place to implement global functions like NYPLDoNotLocalize?

Reply:
Let's make `NYPLLocalization.m`  and `.h` with a global `NYPLLocalizationNotNeeded` function.